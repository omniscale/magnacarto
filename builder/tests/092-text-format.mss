@elev_size: 6;

#labels_format {
    text-name: [name] + "\n" <Format size=@elev_size> [elev] + "m"</Format>;
    text-size: 8;
}

