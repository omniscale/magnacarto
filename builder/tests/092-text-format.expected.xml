<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="labels_format" filter-mode="first">
    <Rule>
      <TextSymbolizer size="8">[name] + &#34;\n&#34; <Format size="6"> [elev] + &#34;m&#34; </Format></TextSymbolizer>
    </Rule>
  </Style>
  <Layer name="labels_format" srs="" status="off">
    <StyleName>labels_format</StyleName>
  </Layer>
</Map>